README.md

```diff
@@ -19,8 +19,8 @@ state.
 
 First make sure you have the following environment variables defined:
 ```
-AWS_ACCESS_KEY_ID           # Access Key for the aws account that should be shredded
-AWS_SECRET_ACCESS_KEY       # Secret Key for the aws account that should be shredded
+AWS_ACCESS_KEY_ID           # IAM account that has permission to shred the accounts
```

Comment:
This description is not quite accurate; This var holds the Access Key ID of an Access Key pair.

Reply:
For more information, here are the docs around Access Keys https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html